tools/sggolines/tools.go

```diff
@@ -0,0 +1,28 @@
+package golines
+
+import (
+	"context"
+	"os/exec"
+
+	"go.einride.tech/sage/sg"
+	"go.einride.tech/sage/sgtool"
+)
+
+const (
+	name    = "golines"
+	version = "v0.12.2"
+)
+
+// Command returns an [*exec.Cmd] for golines.
+func Command(ctx context.Context, args ...string) *exec.Cmd {
+	sg.Deps(ctx, PrepareCommand)
+	return sg.Command(ctx, sg.FromBinDir(name), args...)
+}
+
+func PrepareCommand(ctx context.Context) error {
+	_, err := sgtool.GoInstall(ctx, "github.com/segmentio/golines", version)
```

Comment:
IMO since golines publishes pre-built artifacts let's use those:
https://github.com/segmentio/golines/releases

Reply:
(faster to install, especially on GitHub Actions when it's fetched from within the same server rack :smile:)

Reply:
Fixed... but I believe github is having issues updating the PR: https://githubstatus.com

Reply:
Ok, now it looks like the PR got updated. 😆